### DISCUSSION TITLE
Github Copilot refuses to login on Visual Studio

### DISCUSSION THREAD

**Discussion by johnkslg** [community/community#76490] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I just bought CoPilot but cant get past this:
```
  GitHub Copilot 1.137.0.0 (v1.137.0.0@2f127d06d)
  GitHub Copilot Agent 1.137.0 (1.137.0)
  WARNING: Copilot may display fewer suggestions. Enable "Show completions on new lines" in IntelliCode advanced settings for more suggestions.
  [DEBUG] [agent] [2023-11-21T17:57:11.935Z] Agent service starting
  [DEBUG] [agent] [2023-11-21T17:57:12.107Z] Telemetry initialized
  Auth Status: NotSignedIn
  NotSignedIn
  NotSignedIn

```
- I am signed into github through Visual Studio on the account I bought copilot for and can do normal git operations on it.  I signed out and back in through VS
- Tried installing the certificate for github.com described here: https://github.com/orgs/community/discussions/50484
- Tried setting the options to 'Accept non-trusted certificates automatically' in options
- Updated visual studio, restarted computer

Still nothing. Sure would be nice if that error message gave some info.  Any ideas?

**Comment by johnkslg** [comment, marked as answer]
I was able to get it working by logging out and back in for Github in Visual Studio -> File -> Account Systems

**Comment by skeddles** [comment]
I removed my github account, now there is no more github section on that page to add my account back...
